tests/test_configuration.py

```diff
@@ -114,8 +114,8 @@ def test_config_file_path():
 
 
 @pytest.fixture
-def config(test_config_file_path):
-    environ = {}
+def config(test_config_file_path, monkeypatch):
+    environ = os.environ.copy()
```

Comment:
I'd _prefer_ not to do this, because I'm always worried about tests which secretly rely on some env var being present without explicitly requiring it -> is this necessary for the tests to pass?

Reply:
No, I can update tasks to avoid this!